### ISSUE TITLE
[TECH DEBT] Purge community extensions from 3008 (master) branch.

### ISSUE THREAD

**Issue by dwoz** [saltstack/salt#65970]
The core team will no longer maintain modules deemed as community maintained. Purge community extensions from 3008 (master) branch.

**Comment by zliang-akamai** [comment]
Hi @dwoz, is there any new way to we can save/maintain those extensions?

**Comment by lee-harmonic** [comment]
is this the cause of #69018? (The removal of reclass pillar support, not the rc1 released in the stable repo)

**Comment by lkubb** [comment]
Yes, affected modules are removed from Salt core. But for many, community extensions have been released already. If a module you rely on still does not have one, you can create an extension and distribute it inside the community organization (https://github.com/salt-extensions).

For more details on this move, see https://saltproject.io/blog/the-great-salt-module-migration/ and https://salt-extensions.github.io/salt-extension-copier/ref/concepts.html.
